### ISSUE TITLE
Bad name changing

### ISSUE THREAD

**Issue by jcornill** [Kerosene2000/atom-header-42#4]
When another user save a file the name and mail from the user who create the file is changing by the name and mail from the person who save the file.
Example:
loginA create the file and loginB save this file
"By: loginA <loginA@student.42.fr>" changed to "By: loginB<loginB@student.42.fr>"

**Comment by jcornill** [comment]
Your header:
```
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   ldi.c                                              :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jcornill <jcornill@student.42.fr>          +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2016/04/25 17:31:20 by stoussay          #+#    #+#             */
/*   Updated: 2016/05/18 14:56:36 by jcornill         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */
```
Normal header:
```
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   ldi.c                                              :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: stoussay <stoussay@student.42.fr>          +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2016/04/25 17:31:20 by stoussay          #+#    #+#             */
/*   Updated: 2016/05/18 14:56:36 by jcornill         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */
```

**Comment by Kerosene2000** [comment]
fixed by 9e38e2c
